src/main/java/org/mozilla/magnet/net/scanner/geolocation/ScannerGeolocation.java

```diff
@@ -0,0 +1,116 @@
+package org.mozilla.magnet.net.scanner.geolocation;
```

Comment:
nit:
Can we move this class outside the org.mozilla.magnet.net package?

Maybe org.mozilla.magnet.gps? Does that work for you?